signer/src/stacks/contracts.rs

```diff
@@ -1009,13 +1009,13 @@ impl AsContractCall for RejectWithdrawalV1 {
 pub struct RotateKeysV1 {
     /// The new set of public keys for all known signers during this
     /// PoX cycle.
-    new_keys: BTreeSet<PublicKey>,
+    pub new_keys: BTreeSet<PublicKey>,
```

Comment:
Now we need to deserialize it from (potentially) arbitrary values. We could add custom methods to avoid making those `pub`.